Source/Workers/createVerticesFromQuantizedTerrainMesh.js

```diff
@@ -94,6 +104,18 @@ define([
         var indexBuffer = IndexDatatype.createTypedArray(quantizedVertexCount + edgeVertexCount, indexBufferLength);
         indexBuffer.set(parameters.indices, 0);
 
+        var occludeePointInScaledSpace;
+        var orientedBoundingBox;
+        var boundingSphere;
+
+        if (exaggeration > 1.0) {
```

Comment:
Also, we should probably have a comment here that says the bounding volume has to be recomputed since the tile payload assumes no exaggeration.